internal/cltest/cltest.go

```diff
@@ -57,7 +58,7 @@ const (
 
 var storeCounter uint64
 
-const MinimumContractPayment = 100
+var MinimumContractPayment = assets.NewLink(100)
```

Comment:
Sure, but let's make it a private since it's only used in `cltest`: `minimumContractPayment`

Reply:
@dimroc cool. Done.